tests/benchmarks/micro_benchmarks/test_flops_performance.py

```diff
@@ -0,0 +1,88 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+"""Tests for flops-perf benchmark."""
+
+import os
+import stat
+from pathlib import Path
+
+from superbench.benchmarks import BenchmarkRegistry, Platform, BenchmarkType
+
+
+def test_flops_performance_cuda():
+    """Test flops-perf benchmark."""
+    benchmark_name = 'flops-perf'
+    (benchmark_class,
+     predefine_params) = BenchmarkRegistry._BenchmarkRegistry__select_benchmark(benchmark_name, Platform.CUDA)
+    assert (benchmark_class)
+
+    benchmark = benchmark_class(
+        benchmark_name, parameters='--num_warmup 200 --n 1024 --k 512 --m 2048 --precision FP32 TF32_TC FP16_TC'
+    )
+
+    # Create fake binary file just for testing.
+    binary_path = os.path.join(os.getenv('SB_MICRO_PATH', '/usr/local'), 'bin')
```

Comment:
it's better to create the fake file in `/tmp` (e.g. `tempfile.mkdtemp()`)and clean it after test, writing to `/usr/local` may cause local tests fail or break local environment. The env can also be set to tmp path during this specific test instead of using a global env.

These preparations (env, tmp path, fake file) are called *test fixture*, which should be created in [`setUp`](https://docs.python.org/3/library/unittest.html#unittest.TestCase.setUp) before each test case and cleaned in [`tearDown`](https://docs.python.org/3/library/unittest.html#unittest.TestCase.tearDown) after each test case in Python's test framework.

Reply:
For our testing, it will not write to /usr/local, we don't have the permission to write to /usr/local too. Currently, it will write to the benchmarks folder of superbench.

Reply:
Have added setUp and tearDown handling.